google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/BulkMutation.java

```diff
@@ -48,10 +47,19 @@ public static BulkMutation create(String tableId) {
     return new BulkMutation(tableId);
```

Comment:
Here we can call the new BulkMutation(TargetId) ctor. And annotate with `@Deprecated`, and add some doc on what the caller should call instead.